composer.json

```diff
@@ -55,7 +55,8 @@
         "psr/cache-implementation": "To use metadata caching.",
         "symfony/cache": "To have metadata caching when using Symfony integration.",
         "symfony/config": "To load XML configuration files.",
-        "symfony/twig-bundle": "To use the Swagger UI integration."
+        "symfony/twig-bundle": "To use the Swagger UI integration.",
+        "swagger-api/swagger-ui": "^2.2"
```

Comment:
Ok so:
1. check if the swagger-ui vendor exists
2. print a warning with a link to the docs

About the `SwaggerUiAction`, should we print a different page if the UI is not available?